src/main/java/ch/njol/skript/structures/StructEvent.java

```diff
@@ -45,7 +45,7 @@ public class StructEvent extends Structure {
 
 	@Override
 	@SuppressWarnings("ConstantConditions")
-	public boolean init(Literal<?>[] args, int matchedPattern, ParseResult parseResult, EntryContainer entryContainer) {
+	public boolean init(Literal<?>[] args, int matchedPattern, ParseResult parseResult, @Nullable EntryContainer entryContainer) {
```

Comment:
Line 68 can now throw an NPE.

Reply:
It's not a simple structure, so it can't be null, but i will add an assertion